python/paddle/fluid/tests/unittests/test_activation_op.py

```diff
@@ -1067,28 +578,70 @@ def setUp(self):
         self.attrs = {'beta': beta}
         self.outputs = {'Out': out}
 
-    def test_check_output(self):
-        self.check_output()
-
     def test_check_grad(self):
         if self.dtype == np.float16:
             return
         self.check_grad(['X'], 'Out', max_relative_error=0.008)
 
-    def init_dtype(self):
-        pass
-
 
-class TestFP16Swish(TestSwish):
-    def init_dtype(self):
-        self.dtype = np.float16
+#------------------ Test Fp16 ----------------------
```

Comment:
why merge the float16 tests into one tests?  It is not a standalone tests, and may lead to a longer test runtime if the float16 op increasing.
Just modify the op_test base class, and leave the float16 tests individual.